### ISSUE TITLE
Cannot use ecl/16.1.3 on OSX

### ISSUE THREAD

**Issue by Chream** [roswell/roswell#199]
$ ros use ecl
dyld: Library not loaded: @libdir@/libecl.16.1.dylib
  Referenced from: /Users/izzy/.roswell/impls/x86-64/darwin/ecl/16.1.3/bin/ecl
  Reason: image not found
Error: unable to use 'ecl'

OSX 10.11.6

Seems like the library does exist at least, but I dont know how to lod it correctly.

ls .roswell/impls/x86-64/darwin/ecl/16.1.3/lib
ecl-16.1.3
libecl.16.1.3.dylib
libecl.16.1.dylib -> libecl.16.1.3.dylib
libecl.16.dylib -> libecl.16.1.3.dylib
libecl.dylib -> libecl.16.1.3.dylib

**Comment by snmsts** [comment]
confirmed something changed from 16.1.3
so I'll check how to fix it.

```
ros install ecl/16.1.2
ros use ecl
```

should work.

**Comment by svetlyak40wt** [comment]
I have this issue too.

Probably, the issue should be renamed to "Cannot use ecl/16.1.3 on OSX", to make it more specific?

**Comment by snmsts** [comment]
hm. investigating with ecl via brew

```
~ $ otool -L ~/.roswell/impls/x86-64/darwin/ecl/16.1.3/bin/ecl
/Users/snmsts/.roswell/impls/x86-64/darwin/ecl/16.1.3/bin/ecl:
        @libdir@/libecl.16.1.dylib (compatibility version 16.1.3, current version 0.0.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.0.0)
~ $ otool -L ~/.roswell/impls/x86-64/darwin/ecl/16.1.2/bin/ecl
/Users/snmsts/.roswell/impls/x86-64/darwin/ecl/16.1.2/bin/ecl:
        /Users/snmsts/.roswell/impls/x86-64/darwin/ecl/16.1.2/lib//libecl.16.1.dylib (compatibility version 16.1.2, current version 0.0.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.0.0)
~ $ otool -L /usr/local/bin/ecl
/usr/local/bin/ecl:
	/usr/local/Cellar/ecl/16.1.2/lib//libecl.16.1.dylib (compatibility version 16.1.2, current version 0.0.0)
	/usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1226.10.1)
```

and 

```
~ $ ls -l /usr/local/lib/libecl.dylib
lrwxr-xr-x   1 snmsts         admin          37 2017-01-03 00:00 /usr/local/lib/libecl.dylib -> ../Cellar/ecl/16.1.2/lib/libecl.dylib

~ $ ros	-L ecl/16.1.3 run
dyld: Library not loaded: @libdir@/libecl.16.1.dylib
  Referenced from: /Users/snmsts/.roswell/impls/x86-64/darwin/ecl/16.1.3/bin/ecl
  Reason: Incompatible library version: ecl requires version 16.1.3 or later, but libecl.16.1.dylib provides version 16.1.2
trace/BPT trap: 5
```

from man dyld
```
       DYLD_FALLBACK_LIBRARY_PATH
              This is a colon separated list of directories that contain libraries.  It  is
              used  as  the default location for libraries not found in their install path.
              By default, it is set to $(HOME)/lib:/usr/local/lib:/lib:/usr/lib.
```

```
% DYLD_FALLBACK_LIBRARY_PATH=/Users/snmsts/.roswell/impls/x86-64/darwin/ecl/16.1.3/lib ros -L ecl/16.1.3 run -- --version
ECL 16.1.3
```

worked. I felt it's ECL 16.1.3's bug.

**Comment by snmsts** [comment]
fixed 824c9db
